src/Symfony/Component/Runtime/GenericRuntime.php

```diff
@@ -53,26 +53,31 @@ class GenericRuntime implements RuntimeInterface
      *   debug?: ?bool,
      *   runtimes?: ?array,
      *   error_handler?: string|false,
+     *   envKey?: ?string,
+     *   debugKey?: ?string,
      * } $options
      */
     public function __construct(array $options = [])
     {
-        $debug = $options['debug'] ?? $_SERVER['APP_DEBUG'] ?? $_ENV['APP_DEBUG'] ?? true;
+        $options['envKey'] ?? $options['envKey'] = 'APP_ENV';
```

Comment:
actually, can you please use a local variable instead of modifying the $options array? (same for the other class and the other variable)

Reply:
You mean as a class attribute? If not, I don't know how it could be possible because I need these values in other methods

Reply:
hum right, let's kee as is